### DISCUSSION TITLE
OTA/OTEthernet Example?

### DISCUSSION THREAD

**Discussion by PockyBum522** [khoih-prog/WebServer_WT32_ETH01#8] [Ideas]
I made up an example for OTA over Ethernet with instructions at: 

https://github.com/PockyBum522/WT32_ETH01-OTA-OTEthernet-Example/blob/main/MQTT_And_OTEthernet_Example/MQTT_And_OTEthernet_Example.ino

If you'd like, I can make a PR to add this to the /examples/ folder in your repo.

If you have any suggestions or requests for changes before I do that, let me know.

**Comment by khoih-prog** [comment]
Hi @PockyBum522 

That's great. Please make the PR. Thanks.

**Comment by khoih-prog** [comment]
It'll be great if you also can have time and make **OTA** PRs for many other libraries, such as

1. [Ethernet_Generic](https://github.com/khoih-prog/Ethernet_Generic)
2. [WiFiWebServer](https://github.com/khoih-prog/WiFiWebServer)
3. [WiFiWebServer_RTL8720](https://github.com/khoih-prog/WiFiWebServer_RTL8720)
4. [AsyncHTTPRequest_Generic](https://github.com/khoih-prog/AsyncHTTPRequest_Generic)
5. [AsyncHTTPRequest_Generic](https://github.com/khoih-prog/AsyncHTTPSRequest_Generic)
6. etc.

**Comment by PockyBum522** [comment]
I have made the pull request in this repo. If you are happy with it, I would gladly also add similar example sketches to the other repos.
